src/libraries/Common/tests/TestUtilities/System/Buffers/BoundedMemory.Unix.cs

```diff
@@ -9,10 +9,7 @@ public static unsafe partial class BoundedMemory
     {
         private static UnixImplementation<T> AllocateWithoutDataPopulationUnix<T>(int elementCount, PoisonPagePlacement placement) where T : unmanaged
         {
-            // On non-Windows platforms, we don't yet have support for changing the permissions of individual pages.
-            // We'll instead use AllocHGlobal / FreeHGlobal to carve out a r+w section of unmanaged memory.
-
-            return new UnixImplementation<T>(elementCount);
+            return new UnixImplementation<T>(elementCount, placement);
```

Comment:
@AaronRobinsonMSFT  @tannergooding  - can you review changes to this file as well. This adds page support for unix as well.